app/uk/gov/hmrc/tariffclassificationfrontend/views/templates/application_template.scala.html

```diff
@@ -116,154 +130,189 @@ <h2 id=@headerId>@messages(headerId)</h2>
+                    }
+                    </td>
+                </tr>
+
+
             </table>
 
-            @if(c.application.isBTI){
-                @if(c.application.asBTI.agent.isDefined){
-                    @defining(c.application.asBTI.agent.get){ agentDetails: AgentDetails =>
-                        @sectionHeading("pdf.application.section.applyingFor.heading")
-                        <table>
-                            <tr>
-                                <td>@messages("pdf.application.section.applyingFor.eori")</td>
-                                <td id="agentDetails.eoriDetails.eori">@agentDetails.eoriDetails.eori</td>
-                            </tr>
-                            <tr>
-                                <td>@messages("pdf.application.section.applyingFor.companyName")</td>
-                                <td id="agentDetails.eoriDetails.businessName">@agentDetails.eoriDetails.businessName</td>
-                            </tr>
-                            <tr>
-                                <td>@messages("pdf.application.section.applyingFor.companyAddress")</td>
-                                <td id="agentDetails.eoriDetails.address">
-                                    @agentDetails.eoriDetails.addressLine1 <br/>
-                                    @agentDetails.eoriDetails.addressLine2 <br/>
-                                    @agentDetails.eoriDetails.addressLine3 <br/>
-                                    @agentDetails.eoriDetails.postcode <br/>
-                                    @agentDetails.eoriDetails.country
-                            </td>
-                            </tr>
-                        </table>
-                    }
+            @if(c.application.asBTI.agent.isDefined) {
+                @defining(c.application.asBTI.agent.get) { agent: AgentDetails =>
+                    @sectionHeading("pdf.application.section.applyingFor.heading")
+                <table>
+                    <tr>
+                        <td>@messages("pdf.application.section.applyingFor.details")</td>
+                        <td id="agentDetails.eoriDetails.address">
+                            @agent.eoriDetails.eori <br/>
+                            @agent.eoriDetails.businessName <br/>
+                            @agent.eoriDetails.addressLine1 <br/>
+                            @agent.eoriDetails.addressLine2 <br/>
+                            @agent.eoriDetails.addressLine3 <br/>
+                            @agent.eoriDetails.postcode <br/>
+                            @agent.eoriDetails.country
+                        </td>
+                    </tr>
+                    <tr>
+                        <td> Written authorisation </td>
+                        <td id="agentDetails.letterOfAuthority">
+                        @if(letterOfAuthority.isDefined){@letterOfAuthority.get.fileName}
+                </td>
```

Comment:
Formatting is a bit odd here

Reply:
Same can be achieved with `letterOfAuthority.map(_.fileName)`